pkg/operator/ceph/csi/template/csi-logrotate-sidecar.yaml

```diff
@@ -1,18 +1,22 @@
 args:
   - |
     echo "Starting the csi-logrotate-sidecar"
-    mkdir -p {{ .CsiLogDirPath }}/cephcsi/logrotate-config/{{ .CSILogFolder }};
-    echo '{{ .CsiLogDirPath }}cephcsi/log/{{ .CSILogFolder }}/*.log {
+    mkdir -p {{ .CsiLogDirPath }}/{{
+.CephClusterNamespace }}/cephcsi/logrotate-config/{{ .CSILogFolder }};
+    echo '{{ .CsiLogDirPath }}/{{
+.CephClusterNamespace }}/cephcsi/log/{{ .CSILogFolder }}/*.log {
```

Comment:
Why do we need to change this to use the CephCluster namespace? CSI is for any/all CephClusters, so it seems like a non-namespaced dir would be fine to me.

Reply:
As earlier mentioned for security, 
we should have logging folder per ceph cluster, so to distinguish that we have the namespace sepration.
@BlaineEXE any other options for that?

Reply:
The csi driver is in the namespace of the operator, and not per-ceph-cluster. So perhaps we have the namespace of the rook operator, but not the namespace of a ceph cluster since there could be multiple ceph clusters managed by the same csi driver.

Reply:
Okay I can add namesapce of operator